src/Trixi.jl

```diff
@@ -80,10 +80,16 @@ export initial_condition_constant,
        initial_condition_sedov_blast_wave, initial_condition_medium_sedov_blast_wave,
        initial_condition_blob,
        initial_condition_orszag_tang,
-       initial_condition_rotor
+       initial_condition_rotor,
+       initial_condition_briowu_shock_tube, initial_condition_torrilhon_shock_tube,
+       initial_condition_ryujones_shock_tube, initial_condition_shu_osher,
+       initial_condition_shu_osher_flipped
```

Comment:
I think Hendrik sorted these by "one setup per line", thus
```suggestion
       initial_condition_briowu_shock_tube,
       initial_condition_torrilhon_shock_tube,
       initial_condition_ryujones_shock_tube,
       initial_condition_shu_osher, initial_condition_shu_osher_flipped
```